### PR TITLE
aws-ebs-csi-driver: update to v0.7.0

### PR DESCRIPTION
**Pull request by ipochi**
Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

File: assets/charts/components/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -84,6 +84,3 @@ serviceAccount:
     annotations: {}
   snapshot:
     annotations: {}
-
-# False if you want to use a different storage option, e.g. OpenEBS
-enableDefaultStorageClass: true
```
Comment (invidian) [inline assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: This seems like our change. Should this be moved to component values?

Comment (ipochi) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: Is it our change ?

I didn't see the `XXX Lokomotive specific change` comment.

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: Then I'd move it to component code to avoid confusion.

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: ping @ipochi

Comment (ipochi) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: @invidian This is already part of the templated `values.yaml`

https://github.com/kinvolk/lokomotive/blob/4b4e249506b2ca85a7c897ba64c4ec0bb8116a62/pkg/components/aws-ebs-csi-driver/component.go#L32

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: I thought we just hardcode the value here. Sorry for the confusion. In this case, this should stay here as a documentation and a default value for value used by the Helm chart. It shouldn't be removed.

Comment (ipochi) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: @invidian 

I haven't removed it manually. Its not part of the helm chart.
I suppose it was added by us in the previous version of the chart, but its not present in 0.6.0 version of the upstream chart.

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: Yes, it seems like our addition used here:
```
$ git grep enableDefaultStorageClass
assets/charts/components/aws-ebs-csi-driver/templates/storageclass.yaml:{{- if .Values.enableDefaultStorageClass }}
```

Perhaps we can add `XXX Lokomotive specific change` then.

Actually, it seems entire storage class manifest is our addition. We should upstream it.

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: Opened an issue upstream: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/605.

Comment (invidian) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: And created a PR: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/606

Comment (ipochi) [reply assets/charts/components/aws-ebs-csi-driver/values.yaml:89]: please re-review.

**Review by invidian — APPROVED** [review]
LGTM, thanks @ipochi

**Review by knrt10 — APPROVED** [review]

**Review by iaguis — APPROVED** [review]

**Pull request merged**